ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1593,3 +1594,36 @@ def expand(
         Output Array
     """
     return ivy.current_backend(x).expand(x, shape, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_function
+def ConcatFromSequence(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    axis: int = 0,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
+    Concatenates arrays if axis = 0 and stacks arrays if axis = 1.
+
+    Parameters
+    ----------
+    ary
+        Array input.
+    axis
+        Axis in order to decide whether to concatenate or stack.
+
+    out
+        optional output array, for writing the result to.
```

Comment:
Also, there should be an argument that defines whether or not to stack (you have named it `axis` but I think `new_axis` would make more sense), and an argument that defines which axis to concat/stack on (this could be named `axis`, as in the original onnx function).